tests/runtime/probe

```diff
@@ -76,6 +76,23 @@ EXPECT Offset outside the function bounds \('vfs_read' size is*
 TIMEOUT 5
 WILL_FAIL
 
+NAME kprobe_offset_module
+RUN {{BPFTRACE}} -e 'kprobe:nft_trans_alloc_gfp+0x5 { printf("hit\n"); exit(); }' -c '/usr/sbin/nft add table bpftrace'
+EXPECT hit
+TIMEOUT 5
+REQUIRES lsmod | grep '^nf_tables'
+REQUIRES /usr/sbin/nft --help
```

Comment:
This won't work until #2315 is merged (currently only the second `REQUIRES` is checked). I'll factor that out and submit as a separate PR not to block this.

Reply:
See #2333.